externals.yaml

```diff
@@ -16,4 +16,7 @@ repositories:
   - tasks
   - pipelines
   ignore-versions: [ "v0.1.0" ]
-
+- url: https://github.com/openshift-cnv/openshift-virtualization-pipelines-tasks
```

Comment:
@ksimon1 I opened https://github.com/openshift-cnv/openshift-virtualization-pipelines-tasks and i don't see tasks there or  am i missing something there :thinking:

Reply:
Tasks are part of release https://github.com/openshift-cnv/openshift-virtualization-pipelines-tasks/releases/tag/4.15.0 - see catalog.yaml, resources.tar.gz.
We are using this approach: https://github.com/openshift-pipelines/tektoncd-catalog/blob/main/docs/workflow-provide-your-tekton-resources.md

Reply:
Thanks for the clarification